addons/web/static/src/js/views/abstract_renderer.js

```diff
@@ -8,11 +8,12 @@ odoo.define('web.AbstractRenderer', function (require) {
  */
 
 var mvc = require('web.mvc');
+const { WidgetAdapterMixin } = require('web.OwlCompatibility');
 
 /**
  * @class AbstractRenderer
  */
-return mvc.Renderer.extend({
+return mvc.Renderer.extend(WidgetAdapterMixin, {
```

Comment:
do you really need to make the abstract renderer implement the mixin? Normally, only basic views instantiate field widgets

Reply:
Like this we will not need to refactor if someone want to add owl component in an extension of AbstractRenderer